scheduler/src/cook/components.clj

```diff
@@ -404,10 +402,15 @@
                                                     :progress-sample-interval-ms (* 1000 60 5)}
                            default-uri-config {:cache true
                                                :executable true
-                                               :extract false}]
+                                               :extract false}
+                           executor-uri-configured (and (:uri executor) (not (str/blank? (get-in executor [:uri :value]))))]
+                       (when executor-uri-configured
+                         (log/warn "Executor uri value is missing, the uri config will be disabled" {:executor executor}))
```

Comment:
Should this be `info` instead of `warn`?

Reply:
Done.